### ISSUE TITLE
Move "Offset" to the left of the standard items

### ISSUE THREAD

**Issue by koteyur** [ramya-rao-a/show-offset#2]
Offset value changes frequently. So it shifts the standard elements every time.

I changed
```js
vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right);
```
to this
```js
vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right, 100);
```
and now it works fine.

Maybe it makes sense to make it customizable?

P.S. It would be great to add the "Go to offset" action by clicking on the status bar item... :)

**Comment by CumHoleZH** [comment]
saved replies @robot

**Comment by ramya-rao-a** [comment]
The offset is similar to the column and line number information that is shown in the status bar. And so offset should be on the same side as the column/line number info

> It would be great to add the "Go to offset" action by clicking on the status bar item... :)

That indeed would be cool. PRs are welcome :)

**Comment by koteyur** [comment]
> And so offset should be on the same side as the column/line number info

I suggest this:
![screenshot from 2018-09-10 09-07-34](https://user-images.githubusercontent.com/4014972/45279202-0172be00-b4d9-11e8-96f3-087a8585ffa5.png)

So it is on the same side but more left than Column/Line.

**Comment by ramya-rao-a** [comment]
The above fix is now out in the latest update (0.0.4) of this extension. Thanks!
